src/FixedDataTableCell.js

```diff
@@ -193,57 +192,39 @@ class FixedDataTableCell extends React.Component {
       isHeaderOrFooter,
       visible,
       zIndex,
-      left1,
-      // position,
+      scrollOffsetLeft,
       ...props
     } = this.props;
-    // debugger
-    // var style = {
-    //   height,
-    //   position:'absolute',
-    //   width,
-    //   zIndex,
-    //   visibility: visible ? 'visible' : 'hidden',
-    // };
-
-    // console.log(this._initialRender,"Hello")
-    // if (this.props.isRTL) {
-    //   style.right = props.left;
-    // } else {
-    //   style.left = props.left;
-    // }
-    // FixedDataTableTranslateDOMPosition(
-    //   style,
-    //   -1*(left1),
-    //   0,
-    //   this.props.initialRender,
-    //   this.props.isRTL
-    // );
-
-    // if (this.props.isRTL) {
-    //   style.right =this.props.left+this.props.offsetLeft-left1;
-    // } else {
-    //   style.left = this.props.left+this.props.offsetLeft-left1;
-    // }
+
     var style = {
       height,
+      position: 'absolute',
       width,
+      zIndex,
       visibility: visible ? 'visible' : 'hidden',
     };
 
+    FixedDataTableTranslateDOMPosition(
+      style,
+      this.props.offsetLeft - 1 * scrollOffsetLeft,
+      0,
+      this.props.initialRender,
+      this.props.isRTL
+    );
+
```

Comment:
This function is used for fixing the position of the particular sell .OffsetLeft is the position of a particular cell group from the left .After fixing the position of a cell here we will add left (distance respective to a particular cell group) to it .